libs/input-mapping/tests/Helper/RealDevStorageTableRewriteHelperTest.php

```diff
@@ -678,34 +687,38 @@ function (string $tableId) use (&$tableExistsExpectedParams, &$tableExistsReturn
         );
 
         $inputOptions = $destinations->getTables();
+        $records = $testHandler->getRecords();
         self::assertCount(3, $inputOptions);
-        self::assertCount(3, $testLogger->records);
+        self::assertCount(3, $records);
 
         self::assertSame('in.c-myBucket.my-table', $inputOptions[0]->getSource());
         self::assertSame(456, $inputOptions[0]->getSourceBranchId());
-        $record = array_shift($testLogger->records);
-        self::assertSame('info', $record['level']);
+        /** @var LogRecord $record */
+        $record = array_shift($records);
+        self::assertSame('INFO', $record->level->getName());
         self::assertSame(
             'Using input "in.c-myBucket.my-table" from dev branch "456".',
-            $record['message'],
+            $record->message,
```

Comment:
aha sakra tohle nevím no - to ale zase nebude fungovat se starým monologem ne? Od jaký verze je možný to takhle použít 3 nebo i 2 ? Je to teda jen v testech, tak by možná stačilo příslušnou verzi dát do dev requirements?

Reply:
jo přidal jsem to tam... ve 2.x to takhle použít nejde